src/common/components/dateInput/DateInputGroup.tsx

```diff
@@ -15,19 +15,25 @@ interface Props {
 
 function DateGroup({ children, id = INPUT_GROUP_ID, label, error }: Props) {
   const isInvalid = Boolean(error);
+  const errorId = `${id}-error`;
 
   return (
     <div
       role="group"
       aria-labelledby={id}
+      aria-describedby={errorId}
```

Comment:
This doesn't do much, but I sneaked it in here. The error will be read when the user focuses the group.

Reply:
I didn't notice this would affect anything, tested with voice over + safari / chrome

Reply:
You will have to use the <kbd>ctrl</kbd> + <kbd>opt</kbd> + <kbd>right arrow</kbd> combination so that you'll move to the group. You also have to stay on the group element for a moment for the description to be read. Still, I doubt it helps much in practice.

Reply:
Ahh of course! For some reason when I'm using keyboard I just tab myself through everything 😄